dotnetapp-4.7/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM microsoft/dotnet-framework:4.7 as builder
```

Comment:
Last week https://github.com/Microsoft/dotnet-framework-docker/pull/37 was merged.  These new `build` images should be used here.  It would greatly simplify the Dockerfiles and the time to build them.  Should be as simple as:

```
microsoft/dotnet-framework-build:4.7.1
WORKDIR /app
COPY . ./
RUN ["msbuild.exe", "dotnetapp-4.7.csproj", "/p:Configuration=Release"]
```

This applies to all the samples except 3.5.